cwl/src/main/scala/cwl/package.scala

```diff
@@ -21,7 +21,7 @@ import scala.util.{Failure, Success, Try}
  * It makes heavy use of Circe YAML/Json auto derivation feature and
  * Circe modules that support the Scala libraries shapeless and Refined.
  *
- * The [[https://oss.sonatype.org/service/local/repositories/releases/archive/com/chuusai/shapeless_2.12/2.3.2/shapeless_2.12-2.3.2-javadoc.jar/!/shapeless/Coproduct.html shapeless.coproduct]] feature allows us to specify a large
+ * The [[https://oss.sonatype.org/service/local/repositories/releases/archive/com/chuusai/shapeless_2.12/2.3.2/shapeless_2.12-2.3.3-javadoc.jar/!/shapeless/Coproduct.html shapeless.coproduct]] feature allows us to specify a large
```

Comment:
Not your fault since this is an accurate inclusion of changes in #5632, but those changes were incorrect. That PR was intended to target jaxb-impl but for some reason halfway-"upgraded" this reference to shapeless.

Reply:
I suspect it's doing string matching within scala classes. I was wondering how on earth it was getting so deep into our scala classes and still understanding what was going on!